frontend/language/src/nb.json

```
@@ -1263,9 +1263,13 @@
   "sync_header.validation_completed": "Endringene er validert og kan deles med andre",
   "testing.testing_in_testenv_body": "Her kan du teste appen din fra A til Å i et produksjonslikt testmiljø.",
   "testing.testing_in_testenv_title": "Test i testmiljø",
+  "text_editor.key.error_duplicate": "Denne IDen finnes allerede",
```

Should we try to even more explicit in our message?
```suggestion
  "text_editor.key.error_duplicate": "Det finnes allerede en tekst med denne IDen",